contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -2,62 +2,150 @@ category: application
           performed.
-      address:
+    (NXinstrument):
+      doc: 
+       - |
+        MPES spectrometer
+       - |
+        xref:
+          spec: ISO 18115-1:2023
+          term: 12.58
+          url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:12.58
+      energy_resolution(NXresolution):
         exists: recommended
-        doc: |
-          Full address (street, street number, ZIP, city, country) of the user's
-          affiliation.
-      email:
-        doc: |
-          Email address of the user.
-      orcid:
+        doc:
+        - |
+          Overall energy resolution of the MPES instrument
+        - |
+          xref:
+            spec: ISO 18115-1:2023
+            term: 10.7 ff.
+            url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:10.7
+        - |
+          xref:
+            spec: ISO 18115-1:2023
+            term: 10.24
+            url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:10.24 
+        physical_quantity:
+        type:
+        resolution(NXdata):
+          magnitude(NX_FLOAT):
+            unit: NX_ENERGY
+      device_information(NXfabrication):
         exists: recommended
-        doc: |
-          Author ID defined by https://orcid.org/.
-    (NXinstrument):
-      energy_resolution(NX_FLOAT):
-        unit: NX_ENERGY
-      (NXsource):
+        vendor:
+          exists: recommended
+        model:
+          exists: recommended
+        identifier:
+          exists: recommended
+      source_TYPE(NXsource):
+        
+        # Comment from mpes may workshop:
+        # - There is much more information which can be provided for NXsource
         doc: |
           The source used to generate the primary photons. Properties refer strictly to
           parameters of the source, not of the output beam. For example, the energy of the
           source is not the optical power of the beam, but the energy of the electron beam
           in a synchrotron and so on.
         type:
-          enumeration: [Synchrotron X-ray Source, Rotating Anode X-ray, Fixed Tube X-ray, UV Laser, Free-Electron Laser, Optical Laser, UV Plasma Source, Metal Jet X-ray, HHG laser]
+          enumeration: [Synchrotron X-ray Source, Rotating Anode X-ray, Fixed Tube X-ray, UV Laser, Free-Electron Laser, Optical Laser, UV Plasma Source, Metal Jet X-ray, HHG laser, UV lamp, Monochromatized electron source, other]
+        type_other:
+          exists: optional
+          doc: |
+            Specification of type, may also go to name.
         name:
+          exists: recommended
         probe:
+          exists: optional
           doc: |
             Type of probe. In photoemission it's always photons, so the full NIAC list is
             restricted.
-          enumeration: [x-ray, ultraviolet, visible light]
-      (NXbeam):
+          enumeration: [photon, x-ray, visible light, ultraviolet]
```

Comment:
Now that I think about it, is there any possibility that a pump beam could contain other probes than these four? For example, a pump beam in pumb-probe experiments made up of electrons? If so, we should relax this restriction because with `source_TYPE` we can alsod describe a pump source.

Reply:
Generally, that might be possible I'd say but we decided to have mpes as a photon-in, electron-out description to not broaden the focus of the appdef too much. Also I would argue that then this is not a PES technique anymore.

Reply:
Well, the probe beam would need to be photon-in and the detection electron-out, right? The pump beam could also be an electron excitation, no? But probably, this is not something that exists in the real world..

Reply:
Well, a use case for this would be an electron flood gun... Not sure though if one should describe this with this beam concept.

Reply:
Right, therefore I would leave it open here. 
```yaml
      source_TYPE(NXsource):
        [...]
        doc: |
          A source used to generate photons. Properties refer strictly to parameters of
          the source, not of the output beam. For example, the energy of the source is
          not the optical power of the beam, but the energy of the electron beam
          in a synchrotron and so on.
        [...]
        probe:
          exists: optional
       device_information(NXfabrication):
```

Reply:
Removed probe enumeration, extended `source_TYPE` docstring